rdkit/__init__.py

```diff
@@ -7,17 +11,23 @@
   __version__ = 'Unknown'
   raise
 
-import logging
 logger = logging.getLogger("rdkit")
+
 # if we are running in a jupyter notebook, enable the extensions
 try:
   kernel_name = get_ipython().__class__.__name__
 
   if kernel_name == 'ZMQInteractiveShell':
+    logger.info("Enabling RDKit %s jupyter extensions" % __version__)
     from rdkit.Chem.Draw import IPythonConsole
-    from rdkit.Chem import LogWarningMsg, WrapLogs
-    WrapLogs()  
-    logger.info("Enabling RDKit %s jupyter extensions"%__version__)
-    
 except:
   pass
+
+# Do logging setup at the end, so users can suppress the
+# "enabling jupyter" message at the root logger.
```

Comment:
I don't think we should use the python logger by default.

Reply:
That makes sense (it's the least invasive).  Should we use it for Jupyter, though?  The old version used `WrapLogs()` if it detected Jupyter.

Reply:
I'd rather not make any changes to default behavior as part of this PR.
If we're going to do that, let's discuss it in a separate PR

Reply:
Sounds good.  I'd like to leave it enabled for now, though, so we can make sure all the tests can pass CI with Python logging enabled - once we know it's good I'll revert to the old behavior.  (Unless there's a better way to do this?)